fission/common.go

```diff
@@ -50,3 +55,37 @@ func checkErr(err error, msg string) {
 		fatal(fmt.Sprintf("Failed to %v: %v", msg, err))
 	}
 }
+
+func httpRequest(method, url, body string, headers []string) *http.Response {
+	if method == "" {
+		method = "GET"
+	}
+
+	if method != http.MethodGet &&
+		method != http.MethodDelete &&
+		method != http.MethodPost &&
+		method != http.MethodPut {
+		// Fatal causes some stale data for the calling function, needs improvement
+		fatal(fmt.Sprintf("Invalid HTTP method '%s'.", method))
+	}
+
+	req, err := http.NewRequest(method, url, strings.NewReader(body))
+	checkErr(err, "Error creating reuquest to function")
```

Comment:
checkErr(err, "create HTTP request")

checkErr prints "Failed to <msg>" where msg is the second argument.